src/main/java/io/neonbee/endpoint/raw/RawEndpoint.java

```diff
@@ -88,6 +88,8 @@ public Future<Router> createEndpointRouter(Vertx vertx, String basePath, JsonObj
     static class RawHandler implements Handler<RoutingContext> {
         private static final String CONTENT_TYPE = "Content-Type";
```

Comment:
I know it is not part of the commit, but similar to my lower comment, I would suggest we do the same, rename this to "CONTENT_TYPE_HINT" and move it to Endpoint.java, also we should use the same structure for the value.